sdk/dotnet/Pulumi/Core/InputMap.cs

```diff
@@ -55,7 +55,14 @@ public void Add(string key, Input<V> value)
         {
             var inputDictionary = (Input<ImmutableDictionary<string, V>>)_outputValue;
             _outputValue = Output.Tuple(inputDictionary, value)
-                                 .Apply(x => x.Item1.Add(key, x.Item2));
+                .Apply(x => x.Item1.Add(key, x.Item2));
+        }
+
+        public void Add(InputMap<V> values)
```

Comment:
Exploits existing conversion operators for collection initializers.

Reply:
Again I'd expect this to be called AddRange.